_posts/2025-08-29-this-week-in-rails.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Dark mode on welcome page and more"
+categories: news
+author: Wojtek
+og_image: assets/images/this-week-in-rails.png
+published: true
+date: 2025-08-29
+---
+
+
+Hi, [Wojtek](https://x.com/morgoth85) here. Part of our newsletter team - Claudio, Greg, Vipul, and me will be at the Rails World next week. Greg and Claudio will also be MCing the Lightning Track. Come to say hi and grab some stickers with newsletter logo!
+
+Now let's take a look at this week's changes.
+
+[Dark mode support in the app welcome page](https://github.com/rails/rails/pull/55538)  
+Made by the first time Rails contributor.
+Welcome to the commiters community!
+
+[Slightly speed up Active Job enqueueing with hashes/keyword arguments](https://github.com/rails/rails/pull/55544)  
+The splat in the case/when is not great since it allocates an array each time it is encountered.
+This avoids the allocation, uses a set for the reserved keys, and avoids string allocations for symbol keys.
+About 18% faster when the hash has many string keys.
+
+[Make local cache middleware updatable](https://github.com/rails/rails/pull/55553)  
+If the cache client at *Rails.cache* of a booted application changes, the corresponding mounted middleware needs to update in order for request-local caches to be setup properly. Otherwise, redundant cache operations will erroneously hit the datastore.
```

Comment:
I think we normally wrap code like Rails.cache in backquotes so it looks like code like `Rails.cache` but *Rails.cache* can work too:

<img width="1230" height="928" alt="Scan 2025-08-29 at 12 04 35 AM" src="https://github.com/user-attachments/assets/b4df81f5-a267-4dc6-94b3-d899d4673034" />